packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -780,11 +790,29 @@ class WKUIDelegate extends NSObject {
     WKMediaCaptureType type,
   )? requestMediaCapturePermission;
 
+  /// notifies the host application that the web page
+  /// wants to display a JavaScript alert() dialog.
+  final Future<void> Function(WKJavaScriptPanelData data)?
```

Comment:
These also need to follow the method signature of the corresponding Objc class: https://developer.apple.com/documentation/webkit/wkuidelegate/1537406-webview?language=objc.

Reply:
@bparrishMines 

I appreciate your review.
To follow the method signature, I needed to add request field on WKFrameInfo.
Can you check last commit?
Thanks :)